Xamarin.Forms.Platform.Android/AppCompat/SwitchRenderer.cs

```diff
@@ -0,0 +1,172 @@
+using System;
```

Comment:
why are you adding this file?

Reply:
Merge error. Fixed. This file no longer exists.